### PR TITLE
Backport 53492 - Fix syndic connection when using tcp transport

### PR DESCRIPTION
**Pull request by dwoz**
### What does this PR do?

Fix syndic connection when using tcp transport

### Tests written?

No - Fixing issue in existing test suite

### Commits signed with GPG?

Yes

### CODE REVIEW

File: tests/integration/files/log_handlers/runtests_log_handler.py [hunk 1]
```diff
@@ -35,8 +34,6 @@
 def __virtual__():
     if 'runtests_log_port' not in __opts__:
         return False, "'runtests_log_port' not in options"
-    if six.PY3:
```
Comment (Ch3LL) [inline tests/integration/files/log_handlers/runtests_log_handler.py]: can we revert this for now since we showed it hanging tests on the 2019.2.1 branch?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: tests/integration/files/log_handlers/runtests_log_handler.py (hunk 1 above)
Comment (dwoz) [reply tests/integration/files/log_handlers/runtests_log_handler.py]: Done.

File: tests/integration/files/log_handlers/runtests_log_handler.py [hunk 2]
```diff
@@ -23,7 +23,6 @@
 import msgpack
 
 # Import Salt libs
-from salt.ext import six
```
Comment (Ch3LL) [inline tests/integration/files/log_handlers/runtests_log_handler.py]: just need to re-add this import back in

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Comment by dwoz** [comment]
This PR is no longer needed

**Pull request closed without merging**